wpeverest/user-registration
# UR-1264  Feature -Min max setting for default fields.

### All Submissions:






### Changes proposed in this Pull Request:



Closes # . Previously there was no min/max setting for default fields. This PR introduces min max settings for those fields 

### How to test the changes in this Pull Request:

1.Drag defaults fields and go to advance settings.
2.Verify if min/max settings are working
3.

### Types of changes:




### Other information:




### Changelog entry

>  Feature -Min max setting for default fields.
